settings/CPCT.ini

```diff
@@ -57,6 +57,7 @@ BAF	no
 ANNOTATE_VARIANTS	yes
 KINSHIP	yes
 FINALIZE	yes
+METADATA	yes
```

Comment:
Unlike the other variables in this config section, it is not a pipeline step. Seems like it belongs at the top/bottom of the file or in its own section.

Nitpick: I'm not 100% of the context and potential plans to extend it, but find the "SET" terminology - although used already in the queueing - a little vague. Perhaps SAMPLE_NAME_METADATA or some such thing.